### PR TITLE
Adds a delete column to rh-table

### PR DESCRIPTION
**Pull request by jrit**
@dmaloneycalu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,12 +136,20 @@ Whether a record can be deleted.
...
-*String* | Default: "Edit"
+*String* | Default: ""
 
 The heading of the edit column in `rh-table` when it is visible.
 
 * * *
 
+#### `deleteHeading`
+
+*String* | Default: ""
```
Comment (dmaloneycalu) [inline README.md]: should this and `editHeading` default to an empty string?

Comment (jrit) [inline README.md]: Yeah, the Edit column was documented incorrectly, the default has always been an empty string

**Comment by dmaloneycalu** [comment]
Code LGTM, not seeing any tests for the new column though

**Comment by jrit** [comment]
Thanks, added tests.

Note that I tried installing angular-datetime from npm and couldn't build from it, might be something wrong with the package.

**Pull request merged**